CondFormats/PPSObjects/interface/PPSAlignmentConfig.h

```diff
@@ -87,6 +87,8 @@ struct Binning {
 
   unsigned int n_bins_y_;
   double y_min_, y_max_;
+
+  COND_SERIALIZABLE;

```

Comment:
@ggovi usually it is not allowed to change the a class in CondFormat, but this is a bugfix, is it ok?